build.sbt

```diff
@@ -29,16 +29,13 @@ def scalacOptionsVersion(scalaVersion: String): Seq[String] = {
     //  https://github.com/scala/bug/issues/10047
     CrossVersion.partialVersion(scalaVersion) match {
       case Some((2, scalaMajor: Long)) if scalaMajor < 12 => Seq()
-      case _ => Seq("-Xsource:2.11")
+      case _ => Seq()
     }
   }
 }
 
 def javacOptionsVersion(scalaVersion: String): Seq[String] = {
   Seq() ++ {
-    // Scala 2.12 requires Java 8. We continue to generate
-    //  Java 7 compatible code for Scala 2.11
-    //  for compatibility with old clients.
     CrossVersion.partialVersion(scalaVersion) match {
       case Some((2, scalaMajor: Long)) if scalaMajor < 12 =>
         Seq("-source", "1.7", "-target", "1.7")
```

Comment:
I'd replace this entire method with just:
```scala
def javacOptionsVersion(scalaVersion: String): Seq[String] = Seq("-source", "1.8", "-target", "1.8")
```
`scalaMajor` can never be `< 12`

Reply:
Reduced to default case when major num >= 12